test/specs/chrome.js

```diff
@@ -0,0 +1,49 @@
+import { describe } from '../utils/mochaw'
+import { asyncForEach } from '../utils/async'
+import { supportedLanguages } from '../utils/config'
+import { welcomeScenarios } from './scenarios/welcome'
+import { documentSelectorScenarios } from './scenarios/documentSelector'
+import { documentScenarios } from './scenarios/document'
+import { faceScenarios } from './scenarios/face'
+import { crossDeviceScenarios } from './scenarios/crossDevice'
+import { proofOfAddressScenarios } from './scenarios/proofOfAddress'
+import { modalScenarios } from './scenarios/modal'
+import { navigationScenarios } from './scenarios/navigation'
+
+const options = {
+  screens: [
+    'Camera',
+    'Confirm',
+    'DocumentSelector',
+    'Welcome',
+    'DocumentUpload',
+    'VerificationComplete',
+    'CrossDeviceIntro',
+    'CrossDeviceLink',
+    'CrossDeviceMobileNotificationSent',
+    'CrossDeviceMobileConnected',
+    'CrossDeviceClientSuccess',
+    'CrossDeviceSubmit',
+    'PoaIntro',
+    'PoaDocumentSelection',
+    'PoaGuidance',
+    'Common',
+    'CameraPermissions',
+    'LivenessIntro'
+  ]
+}
+
+describe('Happy Paths', options, ({driver, screens}) => {
```

Comment:
Just one question so I understand it completely. 

If we would like to run only `faceScenarios` on Safari. We'd need to only create `safari.js` file and include `faceScenarios(driver, screens, lang)` in here (+apply the changes in `config.json)?

Reply:
That's correct